acceptance/setup/pre_suite/70_install_released_puppetdb.rb

```diff
@@ -2,13 +2,19 @@
 OLDEST_SUPPORTED_UPGRADE="2.3.8"
 
 if ([:upgrade_oldest, :upgrade_latest].include? test_config[:install_mode] and not test_config[:skip_presuite_provisioning])
-  install_target = test_config[:install_mode] == :upgrade_latest ? 'latest' : OLDEST_SUPPORTED_UPGRADE
+  if test_config[:install_mode] == :upgrade_latest
+    install_target = 'latest'
+    terminus_package = 'puppetdb-termini'
+  else
```

Comment:
is this the `:upgrade_oldest` branch? if so, I'd like this better as an explicit test followed by a bailout on an unexpected value.